lib/guessing_game.js

```diff
@@ -0,0 +1,28 @@
+var pry = require('pryjs');
+function GuessingGame(num){
+  var num = num;
+  var solvedStatus = false
+
+  this.isSolved = function(swap){
+    
+    if(swap === "got it"){
+      solvedStatus = true
+    }
+    return solvedStatus
+  }
+
+  this.guess = function(userGuess) {
```

Comment:
Could we use GuessingGame.prototype.guess for this instead? My thought process is that strategy could extract some logic. Does this not work the way I think it would?